CodeEdit/Localization/be.lproj/Localizable.strings

```diff
@@ -39,3 +39,6 @@
 // Welcome Screen - Workspaces
 "Open a project or file"="Адкрыйце праект або файл";
 "Open an existing project or file on your Mac"="Адкрыйце існуючы праект або файл на вашым Mac";
+
+//Editor Screen
+"CodeEditor can currently not open this file type." = "Зараз CodeEditor не можа адкрыць файл гэтага тыпу.";
```

Comment:
```suggestion
"CodeEdit cannot open this file type." = "Зараз CodeEditor не можа адкрыць файл гэтага тыпу.";
```

Reply:
Cannot is not something you want to say, whereas 'can currently not' suggests that it will eventually support the file type preview if it makes sense?

Reply:
"can currently not" does not make grammatical sense in English, it would instead be "currently cannot". I don't know if we want to imply that we will one day support file types that we might never support.

Reply:
Make sense... Idk our English is very different from most countries. But makes sense and I get you. The requested changes were made

Reply:
Also, technically we are opening a file of that specific file type rather than opening the file type itself. So it might say something like "CodeEdit cannot open this file because it's file type is not supported."